pkg/operator/controller/credentials-secret/controller_test.go

```diff
@@ -131,6 +135,106 @@ func TestReconcile(t *testing.T) {
 			inputRequest:    testRequest(),
 			expectedResult:  reconcile.Result{},
 		},
+		{
+			name:            "Target secret has expected keys for Infoblox provider",
+			existingObjects: []runtime.Object{testInfobloxExtDNSInstance(), testInfoBloxSrcSecret(), testInfoBloxTargetSecret()},
+			inputConfig:     testConfig(),
+			inputRequest:    testRequest(),
+			expectedResult:  reconcile.Result{},
+			expectedEvents: []test.Event{
+				{
+					EventType: watch.Added,
```

Comment:
If the target secret has all expected keys and the same value - we are supposed to not see any events. `Added` event means that the target secret was created.